tests/HasNameTest.php

```
@@ -66,9 +64,9 @@ public function testReturnNewInstanceWhenSettingNameAttribute(): void
      */
     #[DataProviderExternal(NameProvider::class, 'values')]
     public function testSetNameAttributeValue(
-        string|UnitEnum|null $name,
+        string|UnitEnum|Stringable|null $name,
         array $attributes,
-        string|UnitEnum $expectedValue,
+        string|Stringable|UnitEnum $expectedValue,
         string $expectedRenderAttributes,
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $expectedRenderAttributes. Keep variable name length under 20. (undefined)

(LongVariable)